sdk/e2e/11_deploy_router_upi_traffic_split_test.py

```diff
@@ -153,7 +155,7 @@ def test_deploy_router_upi_traffic_split():
             variable_pb2.Variable(
                 name="client_id",
                 type=type_pb2.TYPE_STRING,
-                string_value="1",
```

Comment:
although named `treatment` this was actually `control`, if traffic rules malfunction and a control via default route, would had pass this anyway